suite-native/module-trading/src/tradingSlice.ts

```diff
@@ -81,6 +81,13 @@ export const tradingSlice = createSliceWithExtraDeps({
         clearTradeOrderIdToBeOpened: state => {
             state.tradeOrderIdToBeOpened = undefined;
         },
+        buyAssetChanged: state => {
+            state.buy.selectedReceiveAccount = undefined;
+            state.buy.amountLimits = undefined;
```

Comment:
You should also reset quotesRequest, otherwise you'll get a "No offers available..." error message after changing currencies.
```suggestion
            state.buy.amountLimits = undefined;
            state.buy.quotesRequest = undefined;
```

Reply:
Fixed.